client/resources/aria-at-harness.mjs

```diff
@@ -430,55 +448,81 @@ function renderVirtualInstructionDocument(doc) {
         className(['problem-select']),
         id(`cmd-${commandIndex}-problem-checkboxes`),
         legend(rich(unexpected.failChoice.options.header)),
-        ...unexpected.failChoice.options.options.map(failOption =>
-          fragment(
-            input(
-              type('checkbox'),
-              value(failOption.description),
-              id(`${failOption.description}-${commandIndex}`),
-              className([`undesirable-${commandIndex}`]),
-              tabIndex(failOption.tabbable ? '0' : '-1'),
-              disabled(!failOption.enabled),
-              checked(failOption.checked),
-              focus(failOption.focus),
-              onchange(ev =>
-                failOption.change(/** @type {HTMLInputElement} */ (ev.currentTarget).checked)
+        ...unexpected.failChoice.options.options.map(failOption => {
+          const failOptionId = failOption.description
+            .toLowerCase()
+            .replace(/[.,]/g, '')
+            .replace(/\s+/g, '-');
+
+          return div(
+            className([`problem-option-container ${failOption.checked ? 'enabled' : ''}`]),
```

Comment:
[It looks like this utility was designed to ignore falsey values](https://github.com/w3c/aria-at-app/blob/d71b720641808024adeacbf0b868f72cad29af66/client/resources/vrender.mjs#L85-L91), so we can write

```suggestion
            className(['problem-option-container', failOption.checked && 'enabled']),
```

Reply:
Neat! Change done at https://github.com/w3c/aria-at/pull/1016/commits/4e82c6819b36e65188db865ad9dc439ffc3254ba, and will be moved over to here.